tcf_website/views/review.py

```diff
@@ -123,3 +123,19 @@ def get_object(self):  # pylint: disable=arguments-differ
             raise PermissionDenied(
                 "You are not allowed to delete this review!")
         return obj
+
+    def delete(self, request, *args, **kwargs):
+        """Overide DeleteView's delete function to add a message confirming deletion."""
+        # Note: we don't use SuccessMessageMixin because it currently has issues
+        # with DeleteViews.
+        # See:
```

Comment:
To answer your question regarding SO, there are several other places in the codebase where people have linked to SO posts, so I think it should be fine. IMO, more comments are almost always better than less (as it stands currently, tCF definitely needs more documentation...).

However, maybe move the `See:` to line 130 (which would move the link to 131) so we don't have that long stretch of white space.

Reply:
Agreed. Will do.